OctaneSdk/Connector/Authentication/LwssoAuthenticationStrategy.cs

```diff
@@ -164,5 +177,22 @@ public void PrepareRequest(HttpWebRequest request)
             request.CookieContainer.Add(new Cookie(LWSSO_COOKIE_NAME, lwSsoCookie, cookiePath, cookieDomain));
             request.CookieContainer.Add(new Cookie(OCTANE_USER_COOKIE_NAME, octaneUserCookie, cookiePath, cookieDomain));
         }
+
+        public Task<String> GetWorkspaceUser()
+        {
+            return Task.FromResult(credentials.user);
```

Comment:
here the most simple way is to check real implementation of connection info and return user for UserPassConnectionInfo or client_id for APIKeyConnectionInfo